traffic_ops/testing/api_contract/v4/prerequisite_data.json

```diff
@@ -0,0 +1 @@
+{"cdns": {"name": "test631", "domainName": "quest574", "dnssecEnabled": false, "id": null, "lastUpdated": null}}
```

Comment:
this document should be formatted to make it easier to both read and edit.

Also, `null` is not a valid value for either `id` or `lastUpdated`, so is this meant to represent a CDN in a request or a response?

Reply:
This file is meant to represent a CDN request,
I need to have API keys before sending a request. So i've store these id and lastupdated keys as null for validating keys in response.

Reply:
A CDN request doesn't include those keys, though. I don't see anywhere in the code where you're accessing `id` or `lastUpdated` directly, so even if you compared them to the response you'd be comparing `None` to some non-`None` object, which will always fail. I don't think I understand what you're saying.